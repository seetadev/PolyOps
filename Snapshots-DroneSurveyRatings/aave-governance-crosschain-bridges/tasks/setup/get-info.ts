import { task } from 'hardhat/config';
import { getMnemonicSigner } from '../../helpers/wallet-helpers';
import { DRE } from '../../helpers/misc-utils';

import dotenv from 'dotenv';
dotenv.config({ path: '../../.env' });

task('get-info', 'print-chain-data').setAction(async (_, localBRE) => {
  await localBRE.run('set-DRE');
  console.log(`Chain ID: ${DRE.network.config.chainId}`);

  const blockNumber = await DRE.ethers.provider.getBlockNumber();
  console.log(`Current Block Number: ${blockNumber}`);

  let mnemonicSigner = await getMnemonicSigner(1);
  mnemonicSigner = mnemonicSigner.connect(DRE.ethers.provider);
  const balance = await mnemonicSigner.getBalance();
  console.log(
    `Balance of ${await mnemonicSigner.getAddress()}: ${DRE.ethers.utils.formatUnits(balance, 18)}`
  );
});
