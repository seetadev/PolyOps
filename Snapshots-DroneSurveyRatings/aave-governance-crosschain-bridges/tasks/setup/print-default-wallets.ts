import { task } from 'hardhat/config';
import { Signer } from 'ethers';
import {
  getPrivateKeySigner,
  getMnemonicSigner,
  getDefenderSigner,
} from '../../helpers/wallet-helpers';
import { setDRE } from '../../helpers/misc-utils';

task('print-default-wallets', 'Show addresses available per .env secrets').setAction(
  async (_, hre) => {
    await setDRE(hre);
    const pkSigner: Signer | null = getPrivateKeySigner();
    if (pkSigner == null) {
      console.log(
        `Mnemonic default wallet address:               private key not provided or invalid`
      );
    } else {
      const pkAddress = await pkSigner.getAddress();
      console.log(`Private key default wallet address:            ${pkAddress}`);
    }

    const mnemonicSigner: Signer | null = getMnemonicSigner();
    if (mnemonicSigner == null) {
      console.log(
        `Mnemonic default wallet address:               mnemonic not provided or invalid`
      );
    } else {
      const mnemonicAddress = await mnemonicSigner.getAddress();
      console.log(`Mnemonic default wallet address:               ${mnemonicAddress}`);
    }

    const defenderSigner: Signer | null = getDefenderSigner();
    if (defenderSigner == null) {
      console.log(
        `OpenZeppelin Defender default wallet address:  defender credentials not provided or invalid`
      );
    } else {
      const defenderAddress = await defenderSigner.getAddress();
      console.log(`OpenZeppelin Defender default wallet address:  ${defenderAddress}`);
    }
  }
);
