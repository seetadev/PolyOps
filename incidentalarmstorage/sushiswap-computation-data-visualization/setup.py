#!/usr/bin/python
from sugar3.activity import bundlebuilder
bundlebuilder.start("SocialCalcActivity")
