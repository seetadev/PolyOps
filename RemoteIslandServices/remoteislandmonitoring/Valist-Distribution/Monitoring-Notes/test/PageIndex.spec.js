import { createLocalVue, mount } from '@vue/test-utils';
import { setupTest } from '@nuxt/test-utils'
import IndexPage from '@/pages/index.vue';
import flushPromises from 'flush-promises';

import { registerComponents } from './setup';

window.ethereum = {chainId: '0x4'};

describe('Index Page', function () {
  setupTest({
    configFile: 'nuxt.config.js'
  });

  let wrapper;

  beforeAll(async function () {
    const localVue = createLocalVue();
    registerComponents(localVue);

    // env defined in global setup
    const storePath = `${process.env.buildDir}/store.js`;
    const NuxtStore = await import(storePath);
    const store = await NuxtStore.createStore();

    // hoisted
    wrapper = mount(IndexPage, {
      localVue: localVue,
      store: store,
      attachTo: document.body
    });
  });


  test('is a Vue instance', async function () {
    await expect(wrapper.vm).toBeTruthy();
  });

  test('renders correctly', async function () {
    await expect(wrapper.element).toMatchSnapshot();
  });

  test('connects to metamask', async function () {
    const metamaskButton = wrapper.find('button');
    await expect(metamaskButton.find('span').text()).toMatch('Connect to Tableland');

    await metamaskButton.trigger('click');
    await flushPromises();

    await expect(metamaskButton.find('span').text()).toMatch('Disconnect');
  });
});
