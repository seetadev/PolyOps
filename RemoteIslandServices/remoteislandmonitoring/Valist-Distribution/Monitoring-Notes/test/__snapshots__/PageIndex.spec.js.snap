// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page renders correctly 1`] = `
<div
  class="container mx-auto"
>
  <div
    class="ml-4 mt-8 flex justify-between"
  >
    <div>
      <div
        class="items-center mr-10 mt-4 font-bold text-orbitron text-3xl"
      >
        
        Tableland Todos
      
      </div>
    </div>
     
    <div
      class="mt-4 mr-4"
    >
      <div
        class="flex"
      >
        <button
          class="relative leading-tight font-button border transition-colors duration-150 focus:outline-none shadow-sm border-gray-300 dark:border-gray-600 hover:border-gray-400 dark:hover:border-gray-500 active:border-gray-500 dark:active:border-gray-300 cursor-pointer  px-5 py-2 rounded-button float-right text-poppins"
          type="button"
        >
          <!---->
          <!---->
          <span
            class=""
          >
            <span>
              
            Connect to Tableland
          
            </span>
             
            <!---->
          </span>
          <!---->
        </button>
      </div>
    </div>
  </div>
   
  <hr
    class="border-gray-200 dark:border-gray-800 my-8"
  />
   
  <div
    class="container mx-auto"
  >
    <!---->
     
    <div
      class="justify-center flex"
    >
      <button
        class="relative leading-tight font-button border transition-colors duration-150 focus:outline-none shadow-sm border-gray-300 dark:border-gray-600 hover:border-gray-400 dark:hover:border-gray-500 active:border-gray-500 dark:active:border-gray-300 cursor-pointer  px-5 py-2 rounded-button text-poppins"
        type="button"
      >
        <!---->
        <!---->
        <span
          class=""
        >
          
        Connect to Tableland
      
        </span>
        <!---->
      </button>
    </div>
  </div>
   
  <!---->
   
  <div
    class="fixed z-40 w-full sm:w-96 left-0 top-0"
  >
    <transition-group-stub
      class="inline-block w-full"
      enteractiveclass="transition ease-out-mijin duration-200"
      enterclass="transform -translate-y-3 opacity-0"
      entertoclass="transform translate-y-0 opacity-100"
      leaveactiveclass="transition ease-in-mijin duration-200 absolute w-full"
      leaveclass="transform translate-y-0 opacity-100"
      leavetoclass="transform -translate-y-1/4 opacity-0"
      moveclass="ease-in-out-mijin duration-200"
      tag="ul"
    />
  </div>
</div>
`;
